BrainPortal/app/views/bourreaux/show.html.erb

```diff
@@ -49,34 +49,34 @@
 <%= error_messages_for @bourreau, :header_message => "Server could not be updated." %>
 
 <div class="display_inline_block" style="min-width: 50%">
-  <%= show_table(@bourreau, :edit_condition => @bourreau.has_owner_access?(current_user)) do |t| %>
+  <%= show_table(@bourreau, :as => :bourreau, :edit_condition => @bourreau.has_owner_access?(current_user)) do |t| %>
```

Comment:
The `:as => :bourreau` is correct, but it must be also added to all the other show_table() helpers in the rest of the file too.